spec/models/statistics/total_savings_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe Statistics::TotalSavings do
   describe '#to_s' do
-    context 'there are completed auctions' do
+    context 'there are complete and successful auctions' do
```

Comment:
Can we create an extra auction that is complete but not successful to ensure it isn't counted?